src/CoreWCF.Http/src/CoreWCF/Channels/HttpTransportBindingElement.cs

```diff
@@ -222,6 +224,27 @@ public override T GetProperty<T>(BindingContext context)
             {
                 return (T)(object)new HttpTransportServiceBuilder();
             }
+
+            if (typeof(T) == typeof(IAuthorizationCapabilities))
+            {
+                var binding = context.BindingParameters.OfType<Binding>().FirstOrDefault();
+                if (binding is HttpBindingBase httpBindingBase)
+                {
```

Comment:
Also need code for WSHttpBinding (which has an WSHttpBindingBase base class which I think contains the equivalent of HttpBindingBase)

Reply:
@g7ed6e, ping in case you missed this comment.

Reply:
@mconnew  Please see here https://github.com/CoreWCF/CoreWCF/pull/850/commits/36f9062d236c9a8f528355d9993929bf219e9568. Implemented at WSHttpBinding level which also exposes a Security property that allow HttpClientCredentialType configuration.